package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@eth-optimism/contracts": "^0.0.2-alpha.7",
     "@eth-optimism/core-utils": "^0.1.10",
-    "@eth-optimism/provider": "^0.0.1-alpha.13",
+    "@eth-optimism/provider": "^0.0.3",
```

Comment:
Oh can we actually remove this dependency now that we're doing the raw `.send(..)` calls?